pkg/containerd/events.go

```diff
@@ -0,0 +1,201 @@
+/*
+Copyright © 2022 SUSE LLC
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package containerd handles port binding events from containerd API
+package containderd
```

Comment:
```suggestion
package containerd
```
Maybe you made it to not conflict with `github.com/containerd/containerd`? In that case maybe rename the import instead…